src/drive/web/modules/drive/Toolbar/components/DocumentTypeData.js

```diff
@@ -0,0 +1,263 @@
+  },
+  {
+    id: '19',
+    classification: 'office',
+    label: 'health_book'
+  },
+  {
+    id: '20',
+    classification: 'certificate',
+    label: 'insurance_card'
+  },
+  {
+    id: '21',
+    classification: 'report',
+    categorie: 'health',
+    label: 'prescription'
+  },
+  {
+    id: '22',
+    classification: 'invoicing',
+    categorie: 'health',
+    label: 'health_invoice'
+  },
+  {
+    id: '23',
+    classification: 'certificate',
+    subject: 'car',
+    label: 'registration'
+  },
+  {
+    id: '24',
+    classification: 'certificate',
+    subject: 'car',
+    label: 'car_insurance'
+  },
+  {
+    id: '25',
+    classification: 'invoicing',
+    categorie: 'car',
+    label: 'mechanic_invoice'
+  },
+  {
+    id: '26',
+    classification: 'invoicing',
+    categorie: 'car',
+    label: 'transport_invoice'
+  },
+  {
+    id: '27',
+    classification: 'invoicing',
+    categorie: 'phone',
+    label: 'phone_invoice'
+  },
+  {
+    id: '28',
+    classification: 'invoicing',
+    categorie: 'isp',
+    label: 'isp_invoice'
+  },
+  {
+    id: '29',
+    classification: 'invoicing',
+    categorie: 'energy',
+    label: 'energy_invoice'
+  },
+  {
+    id: '30',
+    classification: 'invoicing',
+    categorie: 'web',
+    label: 'web_service_invoice'
+  },
+  {
+    id: '31',
+    classification: 'contract',
+    subject: 'house',
+    label: 'lease'
+  },
+  {
+    id: '32',
+    classification: 'contract',
+    subject: 'home',
+    label: 'rent_receipt'
+  },
+  {
+    id: '33',
+    classification: 'contract',
+    subject: 'house',
+    label: 'house_insurance'
+  }
+]
+export const categories = [
+  {
+    id: 'cat1',
+    label: 'identity',
+    icon: 'people',
+    file_type_ids: [1, 2, 3, 4, 5, 6]
+  },
+  {
+    id: 'cat2',
+    label: 'family',
+    icon: 'team',
+    file_type_ids: [4, 5, 7, 8, 9, 10, 11]
+  },
+  {
+    id: 'cat3',
+    label: 'work_study',
+    icon: 'company',
+    file_type_ids: [12, 13, 14, 15, 16, 17, 18]
+  },
+  {
+    id: 'cat4',
+    label: 'health',
+    icon: 'heart',
+    file_type_ids: [19, 20, 21, 22]
+  },
+  {
+    id: 'cat5',
+    label: 'home',
+    icon: 'home',
+    file_type_ids: [31, 32, 33, 27, 28, 29]
+  },
+  {
+    id: 'cat6',
+    label: 'transport',
+    icon: 'car',
+    file_type_ids: [6, 23, 24, 25, 26]
+  },
+  {
+    id: 'cat8',
+    label: 'invoice',
+    icon: 'euro',
+    file_type_ids: [27, 28, 29, 26, 30]
+  } /* ,
+  {
+    id: 'cat9',
+    label: 'others',
+    icon: 'dots',
+    file_type_ids: []
+  } */
+]
+
+export const getItemsByCategory = ({ label }) => {
+  const items = []
+  categories.map(category => {
+    if (category.label === label) {
+      category.file_type_ids.map(itemId => {
+        items.push(getItemById(itemId))
+      })
+    }
+  })
+  return items
+}
+
+export const getItemById = id => {
+  let itemToReturn = ''
+  items.map(item => {
+    if (item.id == id) itemToReturn = item
+  })
+  return itemToReturn
+}
```

Comment:
Can be shortened to `lodash.find(items, { id })` :)

Reply:
`items.find(item => item.id == id)` also works